test/reload-source-on-error.test.js

```diff
@@ -104,19 +111,22 @@ QUnit.test('by default, only allows a retry once every 30 seconds', function(ass
   let hlsErrorReloadEvents = 0;
   let hlsErrorReloadCanceledEvents = 0;
 
-  this.player.tech_.on('usage', (event) => {
+  this.player.ready = (callback) => {
```

Comment:
This shouldn't be necessary since you set `this.player.ready` in the `beforeEach`